auto_dev/commands/scaffold.py

```diff
@@ -51,44 +54,108 @@ def validate_address(address: str, logger, contract_name: str = None) -> Optiona
     try:
         return Web3.to_checksum_address(str(address))
     except ValueError as e:
-        name_info = f" for {contract_name}" if contract_name else ""
-        logger.error(f"Invalid address format{name_info}: {e}")
+        name_suffix = f" for {contract_name}" if contract_name else ""
+        logger.error(f"Invalid address format{name_suffix}: {e}")
         return None
 
 
+def validate_abi_version(abi_data):
```

Comment:
Errr not sure that random functions should be put in, especially as i am pretty sure both of these are already implemented?